courses/github-for-devs-self.yml

```diff
@@ -16,7 +16,7 @@ sections:
       - DSS-03_issues
       - DSS-END-02_getting-started
   -
-    title: The GitHub Flow pt. 1: Branching and Committing
+    title: "The GitHub Flow pt. 1: Branching and Committing"
```

Comment:
@1point618 per comment in #141 - let's change this to "The GitHub Workflow: Branching and Committing"